### ISSUE TITLE
Accessibility Documentation Improvements

### ISSUE THREAD

**Issue by dlesaca** [AlaskaAirlines/auro_docs#25]
# General Support Request
As a user that is new to the documentation I would like to understand more of the context and purpose of resources featured [here](https://github.com/AlaskaAirlines/auro_docs/blob/master/src/A11Y.md) through better content organization. 

## Possible Solution
Defining some of the terms outlined in the document could help new users understand the purpose of accessibility.

I think the document should be better divided into sections (i.e. Accessibility, testing, tools, etc.) and include subsections that explain each.

**Comment by dlesaca** [comment]
Closing in favor of #26
